### PR TITLE
Update banner for spring launches

### PR DESCRIPTION
**Pull request by runleonarun**
## What are you changing in this pull request and why?

Linking to the spring launch event in our banner

## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding or removing pages (delete if not applicable):
- [ ] Add/remove page in `website/sidebars.js`
- [ ] Provide a unique filename for new pages
- [ ] Add an entry for deleted pages in `website/vercel.json`
- [ ] Run link testing locally with `npm run build` to update the links that point to deleted pages

### CODE REVIEW

**Review by nghi-ly — APPROVED** [review]
lgtm! thank you!

File: website/docusaurus.config.js [hunk 1]
```diff
@@ -71,14 +71,14 @@ var siteSettings = {
     },
     announcementBar: {
       id: "biweekly-demos",
-      content: "Join our bi-weekly demos and see dbt Cloud in action!",
+      content: "Learn about the latest releases and what's coming next to dbt Cloud",
```
Comment (mirnawong1) [inline website/docusaurus.config.js]: this looks great! should we add the actual date and have a cta or add a more clear action here?

```suggestion
      content: "Join us on May 14th for the dbt Cloud Launch Showcase event - Discover dbt Cloud's latest innovations and watch live demos! ✨ "
```

File: website/docusaurus.config.js [hunk 2]
```diff
@@ -71,14 +71,14 @@ var siteSettings = {
     },
     announcementBar: {
       id: "biweekly-demos",
-      content: "Join our bi-weekly demos and see dbt Cloud in action!",
+      content: "Join us on May 14th for the dbt Cloud Launch Showcase event - Discover dbt Cloud's latest innovations and watch live demos! ✨ "
```
Comment (mirnawong1) [inline website/docusaurus.config.js]:
```suggestion
      content: "Join us on May 14th for the dbt Cloud Launch Showcase event for the latest innovations and live demos! "
```

File: website/docusaurus.config.js [hunk 3]
```diff
@@ -71,14 +71,14 @@ var siteSettings = {
     },
     announcementBar: {
       id: "biweekly-demos",
-      content: "Join our bi-weekly demos and see dbt Cloud in action!",
+      content: "Join us on May 14th for the dbt Cloud Launch Showcase event for the latest innovations and live demos! "
```
Comment (mirnawong1) [inline website/docusaurus.config.js]:
```suggestion
      content: "Join us on May 14th for the dbt Cloud Launch Showcase event. Discover the latest innovations and watch live demos! ",
```

**Pull request merged**